packages/psss/src/utils/period.js

```diff
@@ -14,9 +13,11 @@ import { DUE_ISO_DAY } from '../constants';
 
 /**
  *
- * Convenience utils that build on top of @tupaia/utils
+ * Convenience utils that build on top of getCurrentPeriod
```

Comment:
I guess this was an unwanted replacement

```suggestion
 * Convenience utils that build on top of @tupaia/utils
```